src/main/java/com/minerarcana/floralchemy/proxy/IProxy.java

```diff
@@ -1,5 +1,4 @@
 package com.minerarcana.floralchemy.proxy;
 
 public interface IProxy {
```

Comment:
Sooo a class with zero methods????

Reply:
Swear I copied this from some of your code...